src/objects/Address__c.object

```diff
@@ -166,6 +166,17 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
```

Comment:
@sam-knox Would you mind reviewing these new fields added by the Insights team to the Address object? I believe Lauren Beller has looked at these as well.